perfkitbenchmarker/windows_benchmarks/cluster_boot_benchmark.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2014 Google Inc. All rights reserved.
```

Comment:
This seems to be a copy of the linux version. This is a somewhat trivial benchmark, but we should figure out a way not to duplicate code.